class-wds-react-post-search.php

```diff
@@ -407,10 +407,6 @@ public function search_posts( $request ) {
 			endforeach;
 		endif;
 
-		if ( empty( $results ) ) :
-			return new WP_Error( 'wds-react-post-search-results', apply_filters( 'wds_react_post_search_no_results_text', esc_html__( 'No results found.', 'wds-react-post-search' ) ) );
```

Comment:
I think if we just return a string with  `rest_ensure_response( apply_filters( 'wds_react_post_search_no_results_text', esc_html__( 'No results found.', 'wds-react-post-search' ) ) );`,  instead of a WP_Error object, that should fix the 500 error.